gensrc/thrift/PlanNodes.thrift

```diff
@@ -136,6 +136,13 @@ struct TBrokerScanRangeParams {
 
     // If partition_ids is set, data that doesn't in this partition will be filtered.
     8: optional list<i64> partition_ids
+    
+    // hasExprColumnList is a boolean list
+    // the member of list will be set to true when the column has load expr
+    9: optional list<bool> has_expr_column_list
```

Comment:
make it a set, contains all transfer column ids
And finish convert in backend
rename it to `transform_columns`